clients/common/settings-docs.h.in

```diff
@@ -21,6 +21,8 @@
 #define DESCRIBE_DOC_NM_SETTING_WIRELESS_SEEN_BSSIDS N_("A list of BSSIDs (each BSSID formatted as a MAC address like \"00:11:22:33:44:55\") that have been detected as part of the Wi-Fi network.  NetworkManager internally tracks previously seen BSSIDs. The property is only meant for reading and reflects the BSSID list of NetworkManager. The changes you make to this property will not be preserved.")
 #define DESCRIBE_DOC_NM_SETTING_WIRELESS_SSID N_("SSID of the Wi-Fi network. Must be specified.")
 #define DESCRIBE_DOC_NM_SETTING_WIRELESS_TX_POWER N_("If non-zero, directs the device to use the specified transmit power. Units are dBm.  This property is highly driver dependent and not all devices support setting a static transmit power.")
+#define DESCRIBE_DOC_NM_SETTING_WIRELESS_WAKE_ON_WLAN N_("The NMSettingWirelessWakeOnWLan options to enable. Not all devices support all options. May be any combination of NM_SETTING_WIRELESS_WAKE_ON_WLAN_ANY (0x2), NM_SETTING_WIRELESS_WAKE_ON_WLAN_DISCONNECT (0x4), NM_SETTING_WIRELESS_WAKE_ON_WLAN_MAGIC (0x8), NM_SETTING_WIRELESS_WAKE_ON_WLAN_GTK_REKEY_FAILURE (0x10), NM_SETTING_WIRELESS_WAKE_ON_WLAN_EAP_IDENTITY_REQUEST (0x20), NM_SETTING_WIRELESS_WAKE_ON_WLAN_4WAY_HANDSHAKE (0x40), NM_SETTING_WIRELESS_WAKE_ON_WLAN_RFKILL_RELEASE (0x80), NM_SETTING_WIRELESS_WAKE_ON_WLAN_TCP (0x100) or the special values NM_SETTING_WIRELESS_WAKE_ON_WLAN_DEFAULT (0x1) (to use global settings) and NM_SETTING_WIRELESS_WAKE_ON_WLAN_IGNORE (0x8000) (to disable management of Wake-on-LAN in NetworkManager).")
+#define DESCRIBE_DOC_NM_SETTING_WIRELESS_WAKE_ON_WLAN_PASSWORD N_("If specified, the password used with magic-packet-based Wake-on-WLAN, represented as a MAC address.  If NULL, no password will be required.")
```

Comment:
Could you squash this commit into `libnm-core: add wake-on-wlan configuration items`, otherwise, `make check` will break for the intermediate commits.

Note: you can regenerate the file with `NM_TEST_REGENERATE=1 make check`. Note however, to properly generate all files in `clients/tests/test-client.check-on-disk/`, you have to type `make install` before running the tests (so that localization files are installed in your `--prefix`). Essentially:

```
./configure --prefix=$SOMEWHERE
make && make install
NM_TEST_REGENERATE=1 make -j 10 check
```
(or just don't commit the changes under `clients/tests`).